### PR TITLE
Ryan & Ian refactor

### PR DESCRIPTION
**Pull request by rcwestlake**
@dshendrickson @chelseaskovgaard, thanks for reviewing this

### CODE REVIEW

File: lib/containers/MessagesContainer.jsx [hunk 1]
```diff
@@ -33,31 +33,31 @@ export default class MessagesContainer extends Component {
...
-    const sortedMessages = messages.sort((a, b) => {
-      if (direction === 'down') {
-        return b.id - a.id
-      }
-      if (direction === 'up') {
-        return a.id - b.id
-      }
-      return false
-    })
+    const sortedMessages = messages.sort((a, b) => { return direction === 'down' ? b.id - a.id : a.id - b.id })
```
Comment (rcwestlake) [inline lib/containers/MessagesContainer.jsx:36]: We refactored from an if statement to a one line arrow function and turnary.

File: lib/containers/MessagesContainer.jsx [hunk 2]
```diff
@@ -67,22 +67,22 @@ export default class MessagesContainer extends Component {
 
     const user = this.props.user
 
-    let filteredMessages = filter(messages, m => includes(m.content, filterValue))
-    filteredMessages = filter(filteredMessages, m => includes(m.user.email, userValue))
+    let filteredMessages = this.filterSearch()
+    filteredMessages = this.filterUser(filteredMessages)
```
Comment (rcwestlake) [inline lib/containers/MessagesContainer.jsx:71]: Moved logic from Render to class method

File: lib/containers/MessagesContainer.jsx (hunk 1 above)
Comment (ChelseaSkovgaard) [reply lib/containers/MessagesContainer.jsx:36]: great job condensing from 9 lines to one line

File: lib/containers/MessagesContainer.jsx (hunk 2 above)
Comment (ChelseaSkovgaard) [reply lib/containers/MessagesContainer.jsx:71]: makes sense

**Comment by ChelseaSkovgaard** [comment]
Great refactor! I think you should merge it.

File: lib/containers/MessagesContainer.jsx (hunk 1 above)
Comment (rcwestlake) [reply lib/containers/MessagesContainer.jsx:36]: awesome, thanks

Comment (dshendrickson) [reply lib/containers/MessagesContainer.jsx:36]: Much cleaner

**Comment by rcwestlake** [comment]
@rrgayhart we fixed our scrollbar in this pull request which was just closed.

**Pull request merged**